src/internal/hooks/focus-visible/__tests__/index.test.tsx

```diff
@@ -24,65 +20,64 @@ test('should disable focus by default', async () => {
   { key: 'Control', keyCode: 18 },
   { key: 'Meta', keyCode: 91 },
 ].forEach(key => {
-  test(`should not enable focus when ${key.key} key is pressed`, async () => {
-    const { findByTestId } = render(<Fixture />);
+  test(`should not enable focus when ${key.key} key is pressed`, () => {
+    render(<Fixture />);
     fireEvent.keyDown(document.body, key);
-    expect(await findByTestId('button')).not.toHaveAttribute('data-awsui-focus-visible', 'true');
+    expect(document.body).not.toHaveAttribute('data-awsui-focus-visible');
   });
 });
 
-test(`should enable focus when shift-tab is pressed`, async () => {
-  const { findByTestId } = render(<Fixture />);
+test(`should enable focus when shift-tab is pressed`, () => {
+  render(<Fixture />);
   fireEvent.keyDown(document.body, { key: 'Tab', keyCode: 65, shiftKey: true });
-  expect(await findByTestId('button')).toHaveAttribute('data-awsui-focus-visible', 'true');
+  expect(document.body).toHaveAttribute('data-awsui-focus-visible', 'true');
 });
 
-test('should enable focus when keyboard interaction happened', async () => {
-  const { findByTestId } = render(<Fixture />);
+test('should enable focus when keyboard interaction happened', () => {
+  render(<Fixture />);
   fireEvent.keyDown(document.body);
-  expect(await findByTestId('button')).toHaveAttribute('data-awsui-focus-visible', 'true');
+  expect(document.body).toHaveAttribute('data-awsui-focus-visible', 'true');
 });
 
-test('should disable focus when mouse is used after keyboard', async () => {
-  const { findByTestId } = render(<Fixture />);
+test('should disable focus when mouse is used after keyboard', () => {
+  render(<Fixture />);
   fireEvent.keyDown(document.body);
   fireEvent.mouseDown(document.body);
-  expect(await findByTestId('button')).not.toHaveAttribute('data-awsui-focus-visible');
+  expect(document.body).not.toHaveAttribute('data-awsui-focus-visible');
 });
 
-test('should add listeners only once', async () => {
-  const { findByTestId } = render(
+test('should work with multiple components', () => {
```

Comment:
Do we need this test case at all?

Reply:
This test is very similar to the one below, "should add listeners only once", but it has different assertions. Let's keep both, it does not cost us anything